samples/printing/printing.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import print_function
```

Comment:
Probably don't need this...we can drop support for Python 2.

Reply:
so let's remove all `six`usage ?

Reply:
... and the vendored copy living as `wx/lib/pubsub/py2and3.py`